src/utils/backend.js

```diff
@@ -1,8 +1,17 @@
 import axios from 'axios'
 import store from '../store/store.js'
 
-let base_url = process.env['VUE_APP_BASE_URL']
-const API_VERSION = 'v2.0.0'
+function canReadURLFromEnv () {
+  return !!process.env['VUE_APP_BASE_URL']
+}
+export function setBaseURLWithDefaultOrEnvValue () {
+  const defaultUrl = 'https://cluster.datexis.com'
+  return canReadURLFromEnv() ? process.env['VUE_APP_BASE_URL'] : defaultUrl
+}
+
+let base_url = setBaseURLWithDefaultOrEnvValue()
```

Comment:
Why have you readded those methods?
I think this line `let base_url = process.env['VUE_APP_BASE_URL']` is more than enough.
Our frontend shouldn't have a hardcoded url like this`https://cluster.datexis.com`, once a different person rolls out Kubeportal on their own cluster this becomes obsolete. 

So I am really against bringing those functions back, but maybe I am missing some important point?

Reply:
You can just specify which URL to talk to via an enviromant variable.
It may be a slightly more tiring solution for development, if you want to switch between local Kubeportal and development Kubeportal instance, but I think that is requiered.
![image](https://user-images.githubusercontent.com/58952375/111900472-02362480-8a33-11eb-91aa-d4cc4056f3a7.png)

Reply:
didn't work out on my machine. the initial view works but then with /login I requested on 127.0.0.1... cause I didn't want to debug this issue I just used my static server. if that works on your machine, than we can change it. but i tried again..that fails..